base_partner_merge/views/base_partner_merge.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record model="ir.actions.act_window" id="action_partner_deduplicate">
+        <field name="name">Deduplicate Contacts</field>
+        <field name="res_model">base.partner.merge.automatic.wizard</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">form</field>
+        <field name="target">new</field>
+        <field name="context">{'active_test': False}</field>
+    </record>
+
+    <record model="ir.ui.view" id="base_partner_merge_automatic_wizard_form">
```

Comment:
same as above:
id should be the first attribute in record

Reply:
Done